.github/workflows/quicktest.yml

```diff
@@ -91,6 +86,9 @@ jobs:
           composer-options: --ignore-platform-reqs
           custom-cache-suffix: $(date -u -d "-0 month -$(($(date +%d)-1)) days" "+%F")
 
+      - name: Display PHPCS installed standards
```

Comment:
This doesn't validate anything. It just shows the standards, it doesn't error out if expected standards would not be registered.

Also, this information is already available at the bottom of the "Install Composer dependencies" step as the Composer PHPCS plugin will display it as part of its user facing output.